_partials/self-hosted/_scar-migration.mdx

```diff
@@ -0,0 +1,354 @@
+    }
+    EOF
+    }
+
+    get_auth_token() {
+    authtoken=$(curl -k --location --silent --request POST "${ENDPOINT}/v1/auth/syslogin?setCookie=true" \
+        --header 'Content-Type: application/json' \
+        --data "$(auth_request)" | jq ."Authorization")
+    if [[ "${authtoken}" == null ]]; then
+        echo "Error Logging in. Please check the credentials and URL"
+        exit 1
+    fi
+    }
+
+    update_scar_repo() {
+    response=$(curl -k --write-out '%{http_code}' --silent --output /dev/null --location --request PUT "${ENDPOINT}/v1/system/config/scar" \
+        --header 'Content-Type: application/json' \
+        --header 'Accept: application/json' \
+        --header "Cookie: Authorization=${authtoken}" \
+        --data-raw "$(scar_request)")
+    if [[ "${response}" != "204" ]]; then
+        echo "Error: Unable to update the Scar Repo"
+    else
+        echo "Success Setting the Scar Repo. Waiting for a minute to get the changes syncing"
+        sleep 60
+    fi
+    }
+
+    update_instance_type() {
+    response=$(curl -k --write-out '%{http_code}' --location "${ENDPOINT}/v1/jobs/internal/syncCloudInstanceTypesData/run" \
+        --header 'Content-Type: application/json' \
+        --header 'Accept: application/json' \
+        --header "Cookie: Authorization=${authtoken}")
+    if [[ "${response}" != "204" ]]; then
+        echo "Error: Unable to update the Instance Type"
+    else
+        echo "Started Syncing Storage Type from the palette Service"
+    fi
+    }
+
+    update_storage_type() {
+    response=$(curl -k --write-out '%{http_code}' --location "${ENDPOINT}/v1/jobs/internal/syncCloudStorageTypesData/run" \
+        --header 'Content-Type: application/json' \
+        --header 'Accept: application/json' \
+        --header "Cookie: Authorization=${authtoken}")
+    if [[ "${response}" != "204" ]]; then
+        echo "Error: Unable to update the Storage Type"
+    else
+        echo "Started Syncing Storage Type from the palette Service"
+    fi
+    }
+
+    get_auth_token
+    update_scar_repo
+    update_instance_type
+    update_storage_type
+
+    echo "Wait for 5 minute for the sync process to complete."
+    EOF1
+    ```
+
+10. Grant execution permissions to the `scar-registry-update.sh` script.
+
+    ```shell
+    chmod +x scar-registry-update.sh
+    ```
+
+11. Invoke the script to update the SCAR endpoint. Replace `<cluster-url>` with the address of your {props.edition} instance and
+    `<sys-admin-password>` with the system administrator password.
+
+    ```shell
+    ./scar-registry-update.sh <cluster-url> admin <sys-admin-password>
+    ```
+
+    Consider the following example for reference.
+
+    ```shell hideClipboard
+    ./scar-registry-update.sh https://example.spectrocloud.com admin examplepassword
+    ```
+
+    The following message confirms that the script has completed its tasks.
+
+    ```text hideClipboard
+    Wait for 5 minutes for the sync process to complete.
+    ```
+
```

Comment:
Let's add a sentence stating something like "You have now migrated SCAR to your ......."  just to close out the section.

Reply:
:white_check_mark: